wiki/Community/Project_Loved/en.md

```diff
@@ -87,7 +89,7 @@ There are no strict rules governing which maps a captain must nominate for Loved
 
 ## Website
 
-All of the submitted beatmaps can be seen at the [Loved website](https://loved.sh). 
+The [Loved website](https://loved.sh) serves as the hub for everything related to Project Loved. Here, you can view submitted beatmaps, submit your own, browse past votings, and much more.
```

Comment:
```suggestion
The [Loved website](https://loved.sh) serves as the hub for everything related to Project Loved. Here, you can view submitted beatmaps, submit your own, browse past votings, and much more. It is also used by the [Project Loved Team](/wiki/People/Project_Loved_Team) to handle several miscellaneous tasks when nominating maps.
```

Would specify that the site is used and managed by basically all - if not most - team members at this point.

Reply:
Did so